python/ray/serve/api.py

```diff
@@ -571,3 +577,102 @@ def delete(name: str, _blocking: bool = True):
     """
     client = get_global_client()
     client.delete_apps([name], blocking=_blocking)
+
+
+@PublicAPI(stability="alpha")
+def multiplexed(func=None, num_models_per_replica: int = 0):
+    """Coverts a function or method to a multiplexed function.
+
+    The function can be standalone function or a method of a class. The
```

Comment:
Can you type hint `func` with `Callable[[str], None]`? (I'm assuming the return type is `None`).

Reply:
The return type is not `None`, it's actually `Any` -- let's make this clear @sihanwang41!